test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -24,13 +26,16 @@
 public class TestUtil {
     /**
      * Creates an address book containing the given persons.
-     * @throws DuplicatePersonException if two or more given persons are the same
      */
-    public static AddressBook createAddressBook(Person... persons) throws DuplicatePersonException {
+    public static AddressBook createAddressBook(Person... persons) {
         AddressBook addressBook = new AddressBook();
 
         for (Person person : persons) {
-            addressBook.addPerson(person);
+            try {
+                addressBook.addPerson(person);
+            } catch (DuplicatePersonException e) {
+                e.printStackTrace();
```

Comment:
Shouldn't you throw an error rather than just swallowing it? Having duplicate persons in test data is definitely a programmer error.